01-JS-DOM-APIs/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link href="style.css" rel="stylesheet" type="text/css" >
+    <meta charset="utf-8">
+    <title>first js</title>
+    <script src="code.js">
+
+    </script>
+
+  </head>
+  <body onload="setTimeout(show, 4000);">
+
+  <section>
+    <p> Something </p>
+    <section id="hidden">
+      Hello world!
+    </section>
+    <br><br>
+    <input type="button" value="change for a joke" id="buttonSection" onclick="eventRequest();" />
+  </section>
+
+  <section>
+    <input type="button" value="Ajax" id="buttonSection" onclick="eventOnClick();" />
```

Comment:
try to name event handlers such as *eventOnClick* in a more descriptive way, to make sure the  purpose of that function really clear at a first glance